### ISSUE TITLE
Add license and link to library for easier license compliance

### ISSUE THREAD

**Issue by Calinou** [rqrauhvmra/Tobii#48]
Currently, the files in [`dist/js`](https://github.com/midzer/tobii/tree/production/dist/js) only state the following:

```javascript
/**
 * tobii 2.0.0-beta 
 *
 */
```

I suggest changing it to something like this:

```javascript
/**
 * tobii 2.0.0-beta 
 * Licensed under the MIT license.
 * https://github.com/midzer/tobii
 */
```

The header should also be added to the CSS. This way, people using the library can comply with the license more easily. Thanks in advance :slightly_smiling_face:

**Comment by midzer** [comment]
Hey @Calinou 
Yup, good point. This should be done before v2 release

**Comment by midzer** [comment]
Current dist files contain no comments at all. Is this possible with `microbundle`, @ocean90 ?

**Comment by ocean90** [comment]
No, but you can probably add a [`postbuild`](https://docs.npmjs.com/cli/v7/using-npm/scripts#pre--post-scripts) script running [something like this](https://github.com/keithamus/npm-scripts-example/blob/master/tips/add-text-banner-to-output-file.md).

**Comment by midzer** [comment]
Good advice, awesome :)
We'll do this in the upcoming version.

**Comment by midzer** [comment]
@ocean90 I've spent some time trying to get this to work according to your example.

Too bad, with `postbuild` you can't pipe the banner properly in the file: js content is replaced and processing does not continue for other files.

There is a way to do it in BASH https://www.cyberciti.biz/faq/bash-prepend-text-lines-to-file/ but it is kinda ugly to put the whole banner in `package.json` several times.

Do you wanna try it? Otherwise I would leave this one as "wontfix".

**Comment by viliusle** [comment]
It is not only for license compliance, minified resource should include name, version and URL. Problem, that we have 5 js files in dist folder, do we really need all of them?

**Comment by viliusle** [comment]
Related question, why we have both `dist/tobii.js` and `dist/tobii.min.js`, if both are minified?

p.s. first file is smaller.

**Comment by midzer** [comment]
Maybe @ocean90 has more info about those microbundle dist files

**Comment by midzer** [comment]
`tobii.min.js` has a banner now https://github.com/midzer/tobii/commit/ea8a10c95ee6a13a1fb8222fee721522ba8ecc24
